internal/zinc-core/src/main/scala/sbt/internal/inc/Incremental.scala

```diff
@@ -905,11 +907,16 @@ private final class AnalysisCallback(
   }
 
   def getOrNil[A, B](m: collection.Map[A, Seq[B]], a: A): Seq[B] = m.get(a).toList.flatten
+
   def addCompilation(base: Analysis): Analysis =
     base.copy(compilations = base.compilations.add(compilation))
+
   def addUsedNames(base: Analysis): Analysis = {
-    assert(base.relations.names.size == 0)
-    base.copy(relations = base.relations.addUsedNames(usedNames.result()))
+    assert(base.relations.names.isEmpty)
+    base.copy(
+      relations =
+        base.relations.addUsedNames(UsedNames.fromMultiMap(usedNames.mapValues(_.asScala)))
```

Comment:
```
[error] /home/runner/work/zinc/zinc/internal/zinc-core/src/main/scala/sbt/internal/inc/Incremental.scala:918:79: type mismatch;
[error]  found   : scala.collection.MapView[String,scala.collection.mutable.Set[sbt.internal.inc.UsedName]]
[error]  required: scala.collection.Map[String,scala.collection.Set[sbt.internal.inc.UsedName]]
[error]         base.relations.addUsedNames(UsedNames.fromMultiMap(usedNames.mapValues(_.asScala)))
[error]                                                                               ^
```

Reply:
Oh, good practice for 2.12 vs 2.13 cross building :)